data/10-default.json

```diff
@@ -4,7 +4,17 @@
     "serial": 2018011300,
     "ref-kind": "app",
     "name": "com.endlessm.CompanionAppService",
+    "branch": "eos3",
```

Comment:
I don't know if this branch exists yet?

Reply:
Ah yeah, thinking about it, we cannot naively cherry-pick autoinstall lists between versions if we have the branch explicitly specified.

We'll have to parallel-maintain the auto install file between branches. Which is probably not the worst thing, but something to watch out for.

Reply:
We have two apps repos (actually three but forget the other one), one staging and one prod. The system that promotes apps from staging to prod needs the branch to be "eos3". So we should build and publish this on staging with the eos3 branch, it gets promoted by the release apps job in Jenkins.

Reply:
Well I'm thinking more of branches of the *ostree*

For instance, on eos3 ostree systems you probably want to install from the eos3 branch. On eos4 (whenever that is) you probably want to install from the eos4 branch. Hence, two different autoinstall files.

Reply:
Oh right... all of the OS component git repos get branched for each stable series anyway. So at some point we'd have an eos3.<last> branch of eos-application-tools, and master could be switched to "eos4" (or "stable") ahead of eos4.0 being branched.